addOns/sstiscanner/src/main/resources/org/zaproxy/zap/extension/sstiscanner/resources/Messages.properties

```diff
@@ -0,0 +1,17 @@
+# This file defines the default (English) variants of all of the internationalised messages
+
+sstiscanner.name = Zap-ESUP scanner for SSTI
```

Comment:
ESUP?

Reply:
ZAP-ESUP is the name I gave to the plugin in my master thesis.